tests/test_dcrModel.py

```diff
@@ -165,64 +175,85 @@ def makeDummyWcs(self, rotAngle, pixelScale, crval, flipX=True):
-        lsstAlt = 2663.
-        lsstTemperature = 20.*u.Celsius  # in degrees Celcius
+        lsstLat = -30.244639*u.degree
+        lsstLon = -70.749417*u.degree
+        lsstAlt = 2663.*u.m
+        lsstTemperature = 20.*u.Celsius
         lsstHumidity = 40.  # in percent
         lsstPressure = 73892.*u.pascal
-        lsstWeather = Weather(lsstTemperature.value, lsstPressure.value, lsstHumidity)
-        lsstObservatory = Observatory(lsstLon, lsstLat, lsstAlt)
-        airmass = 1.0/np.sin(elevation.asRadians())
-        era = 0.*radians  # on the meridian
-        zenithAngle = 90.*degrees - elevation
-        ra = lsstLon + np.sin(azimuth.asRadians())*zenithAngle/np.cos(lsstLat.asRadians())
-        dec = lsstLat + np.cos(azimuth.asRadians())*zenithAngle
-        visitInfo = afwImage.VisitInfo(era=era,
-                                       boresightRaDec=afwGeom.SpherePoint(ra, dec),
-                                       boresightAzAlt=afwGeom.SpherePoint(azimuth, elevation),
-                                       boresightAirmass=airmass,
-                                       boresightRotAngle=0.*radians,
-                                       observatory=lsstObservatory,
-                                       weather=lsstWeather
-                                       )
+        loc = EarthLocation(lat=lsstLat,
+                            lon=lsstLon,
+                            height=lsstAlt)
+        airmass = 1.0/np.sin(elevation.asDegrees())
+
+        time = Time(2000., format='decimalyear', scale='tai', location=loc)
+        time += 0.5*u.day  # Add half a day since J2000 is defined at noon
+        if randomizeTime:
+            # Pick a random date and time within a 20-year span
+            time += 20*u.year*self.rng.rand()
+        altaz = SkyCoord(alt=elevation.asDegrees(), az=azimuth.asDegrees(),
+                         unit='deg', obstime=time, frame='altaz', location=loc)
+        obsInfo = makeObservationInfo(location=loc,
+                                      detector_exposure_id=exposureId,
+                                      datetime_begin=time,
+                                      datetime_end=time,
+                                      boresight_airmass=airmass,
+                                      boresight_rotation_angle=Angle(0.*u.degree),
+                                      boresight_rotation_coord='sky',
+                                      temperature=lsstTemperature,
+                                      pressure=lsstPressure,
+                                      relative_humidity=lsstHumidity,
+                                      tracking_radec=altaz.icrs,
+                                      altaz_begin=altaz,
+                                      observation_type='science',
+                                      )
+        makeVisitInfo = MakeRawVisitInfoViaObsInfo()
+        visitInfo = makeVisitInfo.observationInfo2visitInfo(obsInfo=obsInfo)
```

Comment:
This is a static method so it should be enough to do `MakeRawVisitInfoViaObsInfo.observationInfo2visitInfo()`.